services/data/src/react/hooks/stableVariablesHash.test.ts

```diff
@@ -0,0 +1,53 @@
+import { stableVariablesHash } from './stableVariablesHash'
+
+describe('stableVariablesHash', () => {
+    it('sorts objects before hashing', () => {
+        const one = {
+            a: {
+                one: 1,
+                two: 2,
+                three: 3,
+            },
+            b: [1, 2, 3],
+            c: 'c',
+        }
+        const two = {
+            c: 'c',
+            b: [1, 2, 3],
+            a: {
+                three: 3,
+                two: 2,
+                one: 1,
+            },
+        }
+
+        expect(stableVariablesHash(one)).toEqual(stableVariablesHash(two))
+    })
+
+    it('can handle primitives', () => {
+        const one = undefined
+        const two = 'string'
+        const three = 3
+        const four = null
+        const five = true
+
+        expect(stableVariablesHash(one)).toMatchInlineSnapshot(`undefined`)
```

Comment:
Why does this one not have the surrounding `"`?

Reply:
No idea. I guess it's just the way jest serializes the value.